<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmoothingFairing</class>
 <widget class="QDockWidget" name="SmoothingFairing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>198</height>
   </rect>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;adsf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;adf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="windowTitle">
   <string>Smoothing / Fairing</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="Insertion_radio_button">
            <property name="text">
             <string>Insertion</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Removal_radio_button">
            <property name="text">
             <string>Removal</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Brush size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Brush_size_spin_box"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="Set_all_vertices_button">
          <property name="text">
           <string>Set All Vertices as ROI</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Clear_ROI_button">
          <property name="text">
           <string>Clear ROI</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="Show_ROI_check_box">
      <property name="text">
       <string>Show ROI</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Smooth_button">
      <property name="text">
       <string>Smooth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Fair_button">
      <property name="text">
       <string>Fair</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
